docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -278,14 +278,23 @@
         <sect3 id="Installing_CentOS-RedHat-Suse">
           <title>CentOS/Red Hat</title>
 
-          <para>To install the Platform you should have the appropriate
-          permissions to install packages. If you have sudo rights, then you
-          can install the platform using yum.</para>
+          <para>To install the HPCC Systems Platform you should have the
+          appropriate rights and permissions to install packages on your
+          system.</para>
+
+          <para>If you do have the appropriate rights and permissions, then
+          you can install the platform using yum. If you intend to install
+          using yum you should install the epel-release repository beforehand
+          to ensure any dependencies are resolved.</para>
+
```

Comment:
could be tighter by removing lines 281-289 and replacing with this
To install the HPCC Systems Platform you should have the appropriate rights and permissions to install packages on your system. One way to install the platform is using yum.